src/libraries/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -241,31 +238,31 @@ public partial class WindowsIdentity : System.Security.Claims.ClaimsIdentity, Sy
         public System.Security.Principal.TokenImpersonationLevel ImpersonationLevel { get { throw null; } }
         public virtual bool IsAnonymous { get { throw null; } }
         public override bool IsAuthenticated { get { throw null; } }
         public virtual bool IsGuest { get { throw null; } }
         public virtual bool IsSystem { get { throw null; } }
         public override string Name { get { throw null; } }
-        public System.Security.Principal.SecurityIdentifier Owner { get { throw null; } }
+        public System.Security.Principal.SecurityIdentifier? Owner { get { throw null; } }
         public virtual System.IntPtr Token { get { throw null; } }
-        public System.Security.Principal.SecurityIdentifier User { get { throw null; } }
+        public System.Security.Principal.SecurityIdentifier? User { get { throw null; } }
         public virtual System.Collections.Generic.IEnumerable<System.Security.Claims.Claim> UserClaims { get { throw null; } }
         public override System.Security.Claims.ClaimsIdentity Clone() { throw null; }
         public void Dispose() { }
         protected virtual void Dispose(bool disposing) { }
         public static System.Security.Principal.WindowsIdentity GetAnonymous() { throw null; }
         public static System.Security.Principal.WindowsIdentity GetCurrent() { throw null; }
-        public static System.Security.Principal.WindowsIdentity GetCurrent(bool ifImpersonating) { throw null; }
+        public static System.Security.Principal.WindowsIdentity? GetCurrent(bool ifImpersonating) { throw null; }
         public static System.Security.Principal.WindowsIdentity GetCurrent(System.Security.Principal.TokenAccessLevels desiredAccess) { throw null; }
         public static void RunImpersonated(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, System.Action action) { }
+        public static System.Threading.Tasks.Task RunImpersonatedAsync(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, System.Func<System.Threading.Tasks.Task> func) { throw null; }
+        public static System.Threading.Tasks.Task<T> RunImpersonatedAsync<T>(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, System.Func<System.Threading.Tasks.Task<T>> func) { throw null; }
         public static T RunImpersonated<T>(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, System.Func<T> func) { throw null; }
-        public static System.Threading.Tasks.Task RunImpersonatedAsync(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, Func<System.Threading.Tasks.Task> func) { throw null; }
-        public static System.Threading.Tasks.Task<T> RunImpersonatedAsync<T>(Microsoft.Win32.SafeHandles.SafeAccessTokenHandle safeAccessTokenHandle, Func<System.Threading.Tasks.Task<T>> func) { throw null; }
         void System.Runtime.Serialization.IDeserializationCallback.OnDeserialization(object sender) { }
```

Comment:
Hmm, looks like the source generator is omitting the nullable annotation for the `sender` argument. Compare with https://github.com/dotnet/runtime/pull/2347/files#diff-7c0b356a7bada939e2b19732c1c3334dR314 @safern any thoughts?

Reply:
Seems that setting it manually doesn't fix the issue on non-windows machines, on account of auto-generated `*.notsupported.cs` files: https://dev.azure.com/dnceng/public/_build/results?buildId=501665&view=logs&j=0ce95488-260d-5e86-47b9-82bc4e703e0b&t=fc172bf4-4375-570c-6063-be42dda01a5a&l=816